apps/coscrad-frontend/src/store/slices/resources/terms/hooks/use-loadable-terms.ts

```diff
@@ -5,5 +5,6 @@ import { fetchTerms } from '../thunks';
 export const useLoadableTerms = () =>
     useLoadable({
```

Comment:
This `Loadable` behavior is really hard to deal with generically. It's good that we are minimizing the need for "fork joins" by denormalizing our views on the back-end.